<launch>
    <include file="$(find pamaral_object_grasping_pattern_recognition)/launch/keypoints_preprocessing.launch"/>
    
    <node name="dataset_keypoints_preprocessing" pkg="pamaral_object_grasping_pattern_recognition" type="dataset_keypoints_preprocessing.py" output="screen">
        <param name="input_folder" value="$(find data)/dataset/real_time_testing2"/>
        <param name="output_folder" value="$(find data)/preprocessed_dataset/real_time_testing2"/>
    </node>
</launch>